roles/pulp/tasks/main.yaml

```diff
@@ -149,6 +149,12 @@
     port: 24817
     timeout: 300
 
+- name: Wait for Pulp API service to be accessible
```

Comment:
This is a duplicate chunk of code now, this is handled on line 146.

Reply:
Did you want to review #43 instead? This (and #50) were just there to try out things while other PRs were ongoing